### ISSUE TITLE
idlelib/News3.txt for 3.13.0 and backports

### ISSUE THREAD

**Issue by terryjreedy** [python/cpython#112953]
Branch 'main' became 3.13.0a0 as of 3.12.0 beta 1: 2023-05-22
However, unless an patch is not backported to 3.12.b#, idlelib/News3.txt items continue going under
What's New in IDLE 3.12.0 on both main and 3.12 until 3.12.0rc?, 2023-08 or so.  (None for 3.12.0b_.)
Subsequent news items go under
What's New in IDLE 3.13.0 (new header) on main branch
What's New in IDLE 3.12.z (new header) on 3.12 branch
What's New in IDLE 3.11.z (existing) on 3.11 branch
In other words, idlelib News3.txt is handled as if main were branched off as of about 3.x.0rc1.
This is different from the changelog attached to What's New in 3.x.

Release peps -- needed for proposed and actual release dates.
3.11 [PEP-664](https://peps.python.org/pep-0664/)
3.12 [PEP-693](https://peps.python.org/pep-0693/)
3.13 [PEP-719](https://peps.python.org/pep-0719/)

### Linked PRs
* gh-112988
* gh-112990
* gh-112992
* gh-129877
* gh-129878
* gh-129879

**Comment by hugovk** [comment]
The backport PRs are merged, ready to close this issue?

**Comment by terryjreedy** [comment]
No. Since 3.7, each 3.x issue like this covers all IDLE PRs for main, while main is 3.x, that don't include the news item for the now-named idlelib/News3.txt.  In recent years, I have mostly included things with the PR.
#99836
#90654
#86007
#81490
#78343
 #75111
